app/models/comment.rb

```diff
@@ -5,4 +5,13 @@ class Comment < ActiveRecord::Base
   validates :user_id, presence: true
   validates :meetup_id, presence: true
   validates :body, presence: true
+
+  def already_exists?
+    Comment.all.each do |comment|
```

Comment:
Unused block argument - `comment`. You can omit the argument if you don't care about it.<br>Use `find_each` instead of `each`.